service-base/src/main/java/org/eclipse/hono/service/management/device/DeviceManagementService.java

```diff
@@ -42,6 +42,9 @@
      * @return A future indicating the outcome of the operation. The <em>status</em> will be
      *            <ul>
      *            <li><em>201 Created</em> if the device has been registered successfully.</li>
+     *            <li><em>400 Bad Request</em> if the request to register the device is malformed.</li>
+     *            <li><em>401 Unauthorized</em> if the authentication credentials are required, but missing.</li>
+     *            <li><em>403 Forbidden</em> if the operation to register the device is not allowed.</li>
```

Comment:
IMHO we shouldn't list the possible status codes here at all but instead simply refer to the Device Registry Management API docs ...
@ctron @dejanb WDYT?

Reply:
:+1: FMPOV it is more effort to maintain the same information in 2 places and it could easily be not in sync at some point. Now I am also convinced that its better to simply refer to the Device Registry Management API docs and to remove the redundant information from the _JavaDoc_.

Reply:
@sophokles73 The link to the API docs are documented in the _JavaDoc_ using `@see`. In this case I would simply remove the status code part from the _JavaDoc_. WDYT?

Reply:
I would rathe radd a link explicitly to the `@return` description so that the correlation is more explicit